### PR TITLE
fix: typo in clean messages periodical task's logging

### PR DESCRIPTION
**Pull request by bowenliang123**
# Summary

- fix typo in logging of `Cleaned unused dataset` to `Cleaned messages` in `clean_message` periodical task
- remove the unused variable `page`

# Checklist

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] I understand that this PR may be closed in case there was no previous discussion or issues. (This doesn't apply to typos!)
- [x] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.
- [x] I've updated the documentation accordingly.
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

### CODE REVIEW

**Comment by bowenliang123** [comment]
cc @JohnJyong @laipz8200

File: api/schedule/clean_messages.py
Comment (laipz8200) [inline api/schedule/clean_messages.py:1]: If we cannot determine the plan, we should make this task fail.

Comment (bowenliang123) [reply api/schedule/clean_messages.py:1]: OK, the fallback has been removed.

**Comment by bowenliang123** [comment]
Hi @JohnJyong, may I ask for your advice to improve this PR? And what's the main reason for closing this PR?

**Comment by laipz8200** [comment]
Hi @bowenliang123. Sorry for close this PR directly, because this task is related to Cloud services. At the moment, we believe it does not require any changes.

**Comment by bowenliang123** [comment]
Thanks for keeping me informed.

Or may I strip changes and only fix the typo in logging, if you can reopen this PR ? `Cleaned unused dataset ` is not correct in log text anyway.

**Comment by laipz8200** [comment]
> only fix the typo in logging

SGTM.

**Comment by bowenliang123** [comment]
Thanks @laipz8200 . The code and the PR's description have been updated, narrowed to minor fix in logging text and unused variable. May I ask for a second round of review in this , @JohnJyong ?

**Review by laipz8200 — APPROVED** [review]
LGTM

**Comment by laipz8200** [comment]
@JohnJyong and I reviewed this change together. Thank you for your contribution!

**Comment by bowenliang123** [comment]
Cool. I'm always glad to have Dify team's feedback and advices.

**Pull request merged**